src/engine/engine.cpp

```diff
@@ -511,7 +512,7 @@ void Engine::takeBugReport(world::World& world)
   auto img = m_presenter->takeScreenshot();
   img.savePng(m_userDataPath / "bugreports" / dirName / "screenshot.png");
 
-  world.save(m_userDataPath / "bugreports" / dirName / "save.yaml", false);
+  world.save(m_userDataPath / "bugreports" / dirName / "save.yaml", std::nullopt, false);
```

Comment:
See other comment in `World::save`.
```suggestion
  world.save(m_userDataPath / "bugreports" / dirName / "save.yaml", false);
```